doc/design/docker-cp.md

```diff
@@ -119,15 +119,17 @@ The personality code should be fairly straight forward, the goal is to call the
  
  2) depending on the endpoint hit depends on the to/from scenario either way in an *off* scenario we would first get the container object and confirm that it is off. 
  
- 3) Once confirmed that the container is off we will need to immediately block starts(and even wait if one is triggered between the status check and requesting the lockdown). In addition to this, we must block other `cp` operations until the current one is complete. 
+ 3) Once confirmed that the container is off. We will make the request to mount for the import/export operation. 
```

Comment:
From a practical perspective it's likely that vSphere gives us everything we need to reconcile a race between copy and power-on operations. Only one of attach/powerOn will succeed as both will cause the container r/w layer to be locked.

In the case of powerOn winning, we should simply fall back to the online path for the copy. In the case of attach winning we would return the 409.